.github/workflows/build.yml

```diff
@@ -56,6 +60,10 @@ jobs:
         ./run_example.sh --dry-run simple_app_under_oe run_test
         ./run_example.sh --dry-run simple_app_under_oe setup_with_auto_policy_generation_for_OE
 
+        ./run_example.sh --dry-run simple_app_under_gramine
+        ./run_example.sh --dry-run simple_app_under_gramine setup
+        ./run_example.sh --dry-run simple_app_under_gramine run_test
```

Comment:
Added some `--dry-run` tests of the script for this new app. Protects. against future regressions.